src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
you shouldn't reset indentation fot the * selector, change only element you need